modules/capi-creating-infrastructure-resource.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="capi-creating-infrastructure-resource_{context}"]
+= Creating a Cluster API infrastructure resource
+
+You can create a provider-specific infrastructure resource by creating a YAML manifest file and applying it with the {oc-first}.
+
+.Prerequisites
+
+* You have deployed an {product-title} cluster.
+
+* You have enabled the use of the Cluster API.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have obtained the cluster ID value.
+
+* You have created and applied the cluster resource.
+
+.Procedure
+
+. Create a `<infrastructure_resource_file>.yaml` file similar to the following:
++
+--
+[source,yaml]
+----
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
```

Comment:
apiVersion: infrastructure.cluster.x-k8s.io/v1beta2

Reply:
This procedure is meant to be for any platform. @damdo said the `v1beta2` was only AWS right now. 

Should this procedure have some note about "use the right value for your cloud provider" since the cloud providers can diverge?